src/keywords.rs

```diff
@@ -1064,6 +1065,8 @@ pub const RESERVED_FOR_TABLE_ALIAS: &[Keyword] = &[
     Keyword::SAMPLE,
     Keyword::TABLESAMPLE,
     Keyword::FROM,
+    // Reserved for SQL Server cursors
```

Comment:
```suggestion
```
I think we can skip the comment, in case the keyword is used in other contexts in the future